rs/boundary_node/ic_boundary/src/core.rs

```diff
@@ -345,6 +374,97 @@ pub async fn main(cli: Cli) -> Result<(), Error> {
         vec![Box::new(metrics_runner), Box::new(generic_limiter_runner)];
     runners.append(&mut registry_runners);
 
+    // HTTP Logs Anonymization
+    let tracker = match (
+        cli.misc.crypto_config,
+        cli.obs.obs_log_anonymization_canister_id,
+    ) {
+        (Some(crypto_config), Some(log_anonymization_cid)) => {
+            let c = tokio::task::spawn_blocking({
+                let registry_client = registry_client.clone();
+
+                move || {
+                    Arc::new(CryptoComponent::new(
+                        &crypto_config,                          // config
+                        Some(tokio::runtime::Handle::current()), // tokio_runtime_handle
+                        registry_client,                         // registry_client
+                        no_op_logger(),                          // logger
+                        None,                                    // metrics
+                    ))
+                }
+            })
+            .await?;
+
+            let pk = tokio::task::spawn_blocking({
+                let c = c.clone();
```

Comment:
nit: I'd make this `Arc::clone(&c)` to make explicit that this is a cheap (Arc-)clone. (I know the Orchestrator, from which this code is copied, doesn't do that, but it should IMO).

Reply:
I always wondered why use `Arc::clone` vs just `.clone` and this is a very good reason. Thanks 👍 made the change.